documentation/dev/setup-and-guidelines.rst

```diff
@@ -166,6 +166,18 @@ A rolling log file handler is used, so if ``flexmeasures.log`` gets to a few meg
 The default logging level is ``WARNING``. To see more, you can update this with the config setting ``LOGGING_LEVEL``, e.g. to ``INFO`` or ``DEBUG``
 
 
+Mocking an Email Server for Development
+--------------------------------
+
+To handle emails locally during development, you can use MailHog. Follow these steps to set it up:
+
+.. code-block:: bash
+
+   $ docker run -p 8025:8025 -p 1025:1025 --name mailhog mailhog/mailhog
+   $ export MAIL_PORT=1025  # Add this to your local flexmeasures.cfg
+
+Now, emails (e.g., password-reset) are being sent via this local server. Go to http://localhost:8025/.
```

Comment:
End last sentence with " to see all sent emails in a web UI."

Reply:
okay done.